tests/Tests.FeatureManagement/CustomFilterTargeting.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.FeatureManagement;
+using Microsoft.FeatureManagement.FeatureFilters;
+using System;
+using System.Threading.Tasks;
+
+namespace Tests.FeatureManagement
+{
+    class CustomFilterTargeting : IFeatureFilter
```

Comment:
Usually filter names end with Filter. So `CustomTargetingFilter`. Any reason to stray?

Reply:
No great reason. Just wanted to note it's a custom filter- but uses a targeting filter within. Was originally `CustomFilterWithTargeting`.

`CustomTargetingFilter` sounded like it's a different type of targeting. But happy to use that name to be consistent.